### DISCUSSION TITLE
关于视频部分功能建议

### DISCUSSION THREAD

**Discussion by chdesb** [the1812/Bilibili-Evolved#2193] [功能建议 / Ideas]
建议能在视频部分添加”在新标签页打开新视频时取消自动播放“的功能，这样在一次性在后台打开大量视频时会获得更好的体验，不至于乱作一团。辛苦了，谢谢。
Offline v1132.32

**Comment by RingingVolcano** [comment]
![image](https://user-images.githubusercontent.com/72140353/130445100-83aed212-bfe0-4521-9df2-3e7c4794887e.png)
![image](https://user-images.githubusercontent.com/72140353/130445184-cb5e5030-2ebc-4061-b036-dd16124d8afc.png)
关闭官方的自动开播就可以了 理论上是不会自动播放的
